corehq/tabs/tabclasses.py

```diff
@@ -1527,6 +1525,13 @@ def _get_commcare_username(request=None, couch_user=None,
                 'show_in_dropdown': True,
             }
 
+    def _user_fields(self):
```

Comment:
Do we have a guide line to do this in python over the template?

Reply:
I don't follow - do you mean generating the navigation bars?

Reply:
Isn't this replacing?
```
{% if can_add_extra_users and can_access_all_locations %}
  <a class="btn btn-default" href="{% url "user_fields_view" domain %}" id="btn-edit_user_fields">
    <i class="fa fa-edit"></i>
    {% trans "Edit User Fields" %}
    </a>
{% endif %}
```

Reply:
Oh I see - yes essentially this page was accessed via a button inside the mobile workers page, but this PR moves it a level up, so it is now accessed via the sidebar.  Now it's on the same level as the mobile workers page.  The sidebar and the navbar are controlled via this `tabclasses` file that manages the main tree of pages on HQ.